### DISCUSSION TITLE
Bulk editing on mobile

### DISCUSSION THREAD

**Discussion by dead10ck** [community/community#59003] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

I know bulk editing project items was recently released, but I can't seem to figure out if it is possible to do bulk editing on the mobile UI. If I long press an item in the table view, and then long press further down, it seems to highlight multiple rows, but if I then click the drop-down for changing the status field, all other items become unselected. Is there something I'm missing, or is it just not possible in the mobile UI?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 

Great question, thanks for asking. Bulk actions are currently **not** supported on mobile at this time. Feel free to leave us product feedback (in this discussion or by creating a new one) about your request for this including use cases.

**Reply by dead10ck** [reply]
Thanks for the answer! My feedback is that this would be great to have on mobile. Bulk editing is a pretty essential feature for organizing heavy traffic projects, and having the ability to do this on my phone would be great for when I'm not able to sit at a computer.
